src/docs/index.jsx

```diff
@@ -77,6 +77,7 @@ const event = {
   location: 'NYC',
   startDatetime: startDatetime.format('YYYYMMDDTHHmmssZ'),
   title: 'Super Fun Event',
+  filename: 'superfun'
```

Comment:
Since `filename` is now no longer part of the event object, let's update this as well. 

Also, can you make a new section in the docs to describe _how_ to rename the download filename, similar to how there are sections for Custom Button Text and Customized Item List? Once that's done I think we can merge this in. 

Thanks!

Reply:
```
<h2 className={subTitleStyles}>Custom Download Filename</h2>
<AddToCalendarModal
  className={componentStyles}
  linkProps={{
    className: linkStyles,
  }}
  filename="EventInvite"
  event={event}
/>
<CodeSnippet>
  {`
    const AddToCalendarModal = AddToCalendarHOC(Button, CalendarModal);
    ...
    <AddToCalendarModal
      className={componentStyles}
      linkProps={{
        className: linkStyles,
    }}
    event={event}
    filename="EventInvite"
    />
  `}
</CodeSnippet>
```

Reply:
Just did 👌